src/components/main/files/browser/mod.rs

```diff
@@ -22,33 +48,21 @@ pub fn FileBrowser(cx: Scope<Props>) -> Element {
                     state: State::Primary
                 }
             )),
-            Folder {
-                name: String::from("New Folder"),
-                state: State::Secondary,
-                children: 3
-            },
-            Folder {
-                name: String::from("Examples"),
-                state: State::Secondary,
-                children: 12
-            },
-            Folder {
-                name: String::from("Logs"),
-                state: State::Secondary,
-                children: 3941
-            },
-            File {
-                name: String::from("Hello World"),
-                state: State::Secondary,
-                kind: String::from("txt"),
-                size: 0
-            },
-            File {
-                name: String::from("Cache.zip"),
-                state: State::Secondary,
-                kind: String::from("archive/zip"),
-                size: 1
-            }
+            files_sorted.iter().map(|file| {
```

Comment:
Do keep in mind that `Directory::get_items` will return an `Item` that may represent either a `File` or `Directory`. There are functions to check to see if its a file or directory such as `Item::is_file`, `Item::is_directory`, or using `Item::item_type` to get a `ItemType` that may return `ItemType::DirectoryItem` or `ItemType::FileItem` and can be resolved to either one by using `Item::get_file` or `Item::get_directory`. 

So I would suggest the following for now. 

```suggestion
            files_sorted.iter().filter(|item| item.item_type() == ItemType::FileItem).map(|file| {
```

Reply:
Perfect my friend!